setup.sh

```diff
@@ -247,20 +264,61 @@ if [ ! -f ${PW_ENV_FILE} ]; then
         while [ -z "${EMAIL}" ]; do
             read -p 'Email: ' EMAIL
         done
-        read -p 'IP Address (leave blank to scan network): ' IP
+        IP=""
+        # Can we reach 192.168.91.1
+        if test_ip "192.168.91.1"; then
+            IP="192.168.91.1"
+            echo "Found Powerwall Gateway at ${IP}"
+            read -p 'Use this IP? [Y/n] ' response
+            if [[ "$response" =~ ^([nN][oO]|[nN])$ ]]; then
+                IP=""
+            else
+                echo "Congratulations!"
```

Comment:
SETUP: If local mode is selected, the setup script will attempt to find the Powerwall gateway at the default IP (192.168.91.1) and ask the user if they wish to set up the TEDAPI extended device vitals metrics. Otherwise, it will prompt the user for the IP address (or to scan).

This means that the user needs to set up the network routes to the PW Gateway prior to running `setup.sh`.